### PR TITLE
Introduce AdMob banner adapter

### PR DESCRIPTION
**Pull request by ValentinPostindustria**
Make flexible mediation delegate architecture. Rewrite MoPub adapters with new delegate. Fix eamples in rendering test app. Create AdMob banner example. #323

### CODE REVIEW

**Comment by ValentinPostindustria** [comment]
MoPub check after applying new architecture. 
изображение
изображение
изображение
изображение
изображение

File: PrebidMobile/PrebidMobile-admobAdapters/src/main/java/org/prebid/mobile/admob/AdMobBannerMediationUtils.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @Override
+    public void setResponseToLocalExtras(@Nullable BidResponse response) {
+        if (response != null) {
+            extras.putString(PrebidBannerAdapter.EXTRA_RESPONSE_ID, response.getId());
+        }
+    }
+
+    @Override
+    public boolean canPerformRefresh() {
+        return true;
```
Comment (YuriyVelichkoPI) [inline PrebidMobile/PrebidMobile-admobAdapters/src/main/java/org/prebid/mobile/admob/AdMobBannerMediationUtils.java:27]: This method should return false if banner is not visible to the user

File: PrebidMobile/PrebidMobile-mopubAdapters/src/main/java/com/mopub/mediation/MoPubBaseMediationUtils.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            }
+            if (adViewKeywords.length() <= MOPUB_QUERY_STRING_LIMIT) {
+                keywordsManager.setKeywords(adViewKeywords);
+            }
+        }
+    }
+
+    @Override
+    public boolean canPerformRefresh() {
+        return true;
```
Comment (YuriyVelichkoPI) [inline PrebidMobile/PrebidMobile-mopubAdapters/src/main/java/com/mopub/mediation/MoPubBaseMediationUtils.java:55]: Its better to return `false` as a default behavior.

**Review by YuriyVelichkoPI — CHANGES_REQUESTED** [review]
In general looks good. Need to correct behavior related to the autorefresh, please look at the attached comments

**Review by YuriyVelichkoPI — APPROVED** [review]
LGTM

**Pull request merged**